pkg/browsers/console.go

```diff
@@ -281,8 +357,7 @@ func makeDestinationURL(service string, region string) (string, error) {
 		// and this avoids the need to keep the ServiceMap alphabetically sorted when developing Granted.
 		sort.Strings(validServices)
 
-		return "", fmt.Errorf("\nservice %s not found, please enter a valid service shortcut.\nValid service shortcuts: [%s]\n", service, strings.Join(validServices, ", "))
-
+		return "", fmt.Errorf("\nservice %s not found, please enter a valid service shortcut.\nValid service shortcuts: [%s]", service, strings.Join(validServices, ", "))
```

Comment:
👍 on this. Given that we're return an `err` here we don't need to worry about a newline at the end of the statement (I think the original implementation was a typo)

Reply:
This was @jordiup 's work from his original PR.